### DISCUSSION TITLE
Play Structure > Progression Structure > Community

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#73] [Framework Discussions]
# Play Structure > Progression Structure > Community

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
Only after reaching a specific Access Level, you unlock the Clans where, after joining one, you can unlock a specific and time-bound game mode

**Comment by Tomathy-dev** [comment]
Players can integrate in guilds, interacting with the other members, socializing, participating in guild made events or by simply progressing with them through the raids available. Even within guilds, sub groups can form to participate in specific types of content.
